packages/ibm-products/src/components/Guidebanner/GuidebannerElement.tsx

```diff
@@ -16,29 +16,47 @@ import { pkg } from '../../settings';
 const blockClass = `${pkg.prefix}--guidebanner__element`;
 const componentName = 'GuidebannerElement';
 
+interface GuidebannerElementProps {
+  /**
+   * An optional button can be rendered below the description.
+   * This can be a link, button, Coachmark button, etc.
+   */
+  button: ReactNode;
+  /**
+   * Provide an optional class to be applied to the containing node.
+   */
+  className?: string;
+  /**
+   * The description of the element.
+   */
+  description: string | ReactNode;
```

Comment:
I believe `string` is also considered a `ReactNode` so you could just specify `description: ReactNode`
```suggestion
  description: ReactNode;
```